realm/realm-library/src/objectServer/java/io/realm/internal/network/AuthenticateResponse.java

```diff
@@ -132,14 +133,14 @@ private AuthenticateResponse(String serverResponse) {
             if (accessToken == null) {
                 message = "accessToken = null";
             } else {
-                message = String.format("Identity %s; Path %s", accessToken.identity(), accessToken.path());
+                message = String.format(Locale.ENGLISH, "Identity %s; Path %s", accessToken.identity(), accessToken.path());
```

Comment:
I tested following test case to make sure Strings are formatted as expected.

```
@Test
public void formatNonAsciiString() {
    final String testee = String.format(Locale.ENGLISH,
            "format %s, %s and %s.", "σκύλοι", "領域", "ангелы");
    assertEquals("format σκύλοι, 領域 and ангелы.", testee);
}
```
After testing, I realized that the Android documentation recommends to use `Locale.US`.
https://developer.android.com/reference/java/util/Locale.html#default_locale

Should I update this PR to use `Locale.US`?